lib/driver.js

```diff
@@ -215,7 +215,7 @@ class AndroidDriver extends BaseDriver {
     }
     // start UiAutomator
     this.bootstrap = new helpers.bootstrap(this.adb, this.bootstrapPort, this.opts.websocket);
-    await this.bootstrap.start(this.opts.appPackage, this.opts.disableAndroidWatchers);
+    await this.bootstrap.start(this.opts.appPackage, this.opts.disableAndroidWatchers, this.acceptSslCerts);
```

Comment:
Actually, if you just make this `this.opts.acceptSslCerts`, you don't need to make changes in appium/appium. Because caps are merged to `this.opts` (or should be).